src/mudlet.h

```diff
@@ -325,8 +325,8 @@ class mudlet : public QMainWindow, public Ui::main_window
     const QMap<QByteArray, QString>& getEncodingNamesMap() const { return mEncodingNameMap; }
     void refreshTabBar();
     void updateDiscordNamedIcon();
-    // Has to be public as it needs to be called from dlgConnectionProfiles class:
```

Comment:
Not sure it is helpful to remove this comment - it gives advice about something that might not be obvious. IMHO.

Reply:
A quick investigation will fix it.